recipes/tyto/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "tyto" %}
+{% set version = "1.0b3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/SynBioDex/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: c67aa4cb5bfe258c3e6b78a5a5adab1c4b325998e52e7993602309264694bf2e
+  # External dependencies
+  - url: https://github.com/SynBioDex/sbol-owl3/archive/1.0.tar.gz
+    sha256: 2fbb20a398ac7b77d143cfa45085a007f377090c4c7fbd03a0584057cdbd1c37
+    folder: tyto/ontologies/sbol-owl3
+  - url: https://github.com/dissys/sbol-owl/archive/v1.1.6.tar.gz
```

Comment:
Can these two dependencies be their own package? Having them hardcoded here with fixed versions may lead to a stale and broken recipe quite fast as the code evolves.

Reply:
Hi @ocefpaf , I understand but unfortunately I can't package these sub modules too. So I published this tool in bioconda channel because it's easier to handle. I will close this PR. Thank you for your work.